tests/pass/const-addrs.rs

```diff
@@ -8,6 +8,7 @@
 // stack frame, breaking this test.
 //@compile-flags: -Zinline-mir=no
 #![feature(strict_provenance)]
+//@only-miri: expecting that pointer addresses have specific behaviour
```

Comment:
```suggestion
//@only-miri: expecting that pointer addresses have specific behavior
#![feature(strict_provenance)]
```
Please generally keep the `//@` flags together